afrc/src/afrc/Search/components/SearchItemDetails.vue

```diff
@@ -22,16 +25,27 @@ watch(resultSelected, () => {
 
 async function getData() {
     const resp = await fetchResourceData(resultSelected.value);
+    const imageResourceids = resp.resource["Digital Reference"]?.map((tile: { [key: string]: any; }) => // eslint-disable-line @typescript-eslint/no-explicit-any
+        tile["Digital Source"]["resourceId"]); 
+    const accessionNumber = resp.resource['Identifier']?.find((identifier: {[key: string]: any;}) => // eslint-disable-line @typescript-eslint/no-explicit-any
+        identifier["Identifier_type"]["@display_value"]==="Accession Number"); 
+
+    acquisitions.value = resp.resource["Addition to Collection"]?.map((tile: { [key: string]: any; }) => // eslint-disable-line @typescript-eslint/no-explicit-any
+        ({ 
+            "person": tile["Addition to Collection_carried out by"]["@display_value"],
+            "date": tile["Addition to Collection_time"]["Addition to Collection_time_begin of the begin"]["@display_value"],
+            "details": tile["Addition to Collection_Statement"]?.map((statement: {[key: string]: any;}) => // eslint-disable-line @typescript-eslint/no-explicit-any
+                (statement["Addition to Collection_Statement_content"]["@display_value"])).join(" ") 
+        }));
```

Comment:
add optional chaining to allow forother  parts of display  to render properly
```suggestion
        ({ 
            "person": tile?.["Addition to Collection_carried out by"]?.["@display_value"],
            "date": tile?.["Addition to Collection_time"]?.["Addition to Collection_time_begin of the begin"]["@display_value"],
            "details": tile?.["Addition to Collection_Statement"]?.map((statement: {[key: string]: any;}) => // eslint-disable-line @typescript-eslint/no-explicit-any
                (statement?.["Addition to Collection_Statement_content"]?.["@display_value"])).join(" ") 
        }));
```

Reply:
Ah - I thought the optional chaining in the `?.map` would ensure there was a tile, but I guess not if you had a parent tile with no children.

Reply:
Ok - that should be fixed now.